src/main/resources/default/assets/javascript/multiLanguageField.js

```diff
@@ -28,23 +30,32 @@ function MultiLanguageField(options) {
 
     this._wrapper = document.getElementById(this.wrapperId);
     this._input = this._wrapper.querySelector('.mls-input');
-    this._hiddenInputs = this._wrapper.querySelector('.mls-hidden-inputs');
-    this._modal = document.getElementById(this.modalId);
-
-    this._modalInputs = this._modal.querySelector('.mls-modal-inputs');
-    this._modalPlaceholder = this._modal.querySelector('.mls-modal-placeholder');
 
     const me = this;
 
-    $(me._modal).on('hidden.bs.modal', function () {
-        me.updateHiddenFields();
-        me.updateOuterInputField();
-    })
+    if (!this.multiline) {
+        this._hiddenInputs = this._wrapper.querySelector('.mls-hidden-inputs');
+        this._modal = document.getElementById(this.modalId);
+        this._modalInputs = this._modal.querySelector('.mls-modal-inputs');
+        this._modalPlaceholder = this._modal.querySelector('.mls-modal-placeholder');
+
+        $(me._modal).on('hidden.bs.modal', function () {
```

Comment:
Ich weiß das hab ich vergessen, aber könntest du evtl bei gelegenheit hier n line comment drüber packen a la `have to use jquery here as bootstrap modals only trigger jquery events`? Wär super

Reply:
Ist mit dem nächsten Commit gefixt